workflow-templates/debian-package-deliver-pipeline.yml

```diff
@@ -0,0 +1,25 @@
+# For official documentation on this workflow, please refer to the link below.
+# https://github.com/brandwatch/bw-workflow-actions/blob/dev/workflows/debian/debian-package-deliver.yaml.md
+
+name: CI Pipeline - Deliver Debian Package
+
+permissions:
+  checks: write
+  contents: read
+
+on:
+  push:
+    branches:
+      - "main"
+
+jobs:
+  deploy-debian-pipeline:
+    name: "Deliver Debian Package"
+    uses: brandwatch/bw-workflow-actions/.github/workflows/debian-package-deliver.yaml@dev
+    with:
+      package-file: package.deb
+      artifactory-directory-and-file: pool/xxx
```

Comment:
A line comment here perhaps to remind them to change this? Perhaps default to `pool/<firstLetterOfDebFile>` is left blank?

Reply:
Ah yeah good point, I've updated this to something more similar to what is on http://ftp.debian.org/debian/pool/, with comments guiding the user. I'm not sure it's absolutely necessary to adhere to all that, but at least they can if they want to

Reply:
LGTM 👍🏻